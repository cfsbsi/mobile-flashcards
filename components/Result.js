import React from 'react'
import styled from "styled-components/native/index";

export default function Result(props) {
    return (
        <MainView>
            <TextStyled>You accepted {props.navigation.state.params.correctPercent}%</TextStyled>
            <TextStyled>of the questions!!</TextStyled>
        </MainView>
    )

}

const MainView = styled.View`
    flex: 1;
    align-items: center;
    justify-content: center;
`

const TextStyled = styled.Text`
   font-size: 40;
   font-weight: bold;
`