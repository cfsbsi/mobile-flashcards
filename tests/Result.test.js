import React from 'react';
import renderer from 'react-test-renderer';
import Result from '../components/Result';

it('renders correctly', () => {
  const state = {
    state: {
      params: {
        correctPercent: 100,
      },
    },
  };
  const tree = renderer.create(<Result navigation={state} />).toJSON();

  expect(tree).toMatchSnapshot();
});
