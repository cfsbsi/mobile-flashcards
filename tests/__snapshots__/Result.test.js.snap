// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 40,
          "fontWeight": "bold",
        },
        undefined,
      ]
    }
  >
    You accepted 
    100
    %
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 40,
          "fontWeight": "bold",
        },
        undefined,
      ]
    }
  >
    of the questions!!
  </Text>
</View>
`;
